Bottom: f7735041583683b606c41fcda88d553dc22b41be
Top:    fdbe904f935b7ec36492132deaba2eeff1199db2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-01 19:38:41 +0000

Refresh of split-drc-routines-out-of-find

---

diff --git a/src/drc.c b/src/drc.c
index 21332c6..ea8d0b3 100644
--- a/src/drc.c
+++ b/src/drc.c
@@ -84,6 +84,7 @@ pcb_drc_violation_free (DrcViolationType *violation)
 }
 
 static GString *drc_dialog_message;
+
 static void
 reset_drc_dialog_message(void)
 {
@@ -95,6 +96,7 @@ reset_drc_dialog_message(void)
       gui->drc_gui->reset_drc_dialog_message ();
     }
 }
+
 static void
 append_drc_dialog_message(const char *fmt, ...)
 {
